sw/device/lib/dif/dif_pwrmgr_unittest.cc

```diff
@@ -894,5 +894,23 @@ TEST_F(IrqTest, RestoreAll) {
   EXPECT_EQ(dif_pwrmgr_irq_restore_all(&pwrmgr_, snapshot), kDifPwrmgrOk);
 }
 
+class AlertTest : public DifPwrmgrInitialized {};
+
+TEST_F(AlertTest, ForceBadArgs) {
+  dif_pwrmgr_alert_t alert = kDifPwrmgrAlertFatalFault;
```

Comment:
nit: can be inlined, or renamed to `valid_alert` to indicate purpose.

Reply:
Fixed.